api/config/config_local.json

```diff
@@ -166,5 +166,9 @@
   },
   "app": {
     "rStudioDescriptorPath": "https:\/\/raw.githubusercontent.com\/DataBiosphere\/terra-app\/39c602e20ba027eb065dcb7690e76f2236ac2848\/apps\/rstudio\/app.yaml"
+  },
+  "termsOfService": {
+    "latestAouVersion": 1,
+    "latestTerraTosTimestamp": "1985-10-26T01:22:33Z"
```

Comment:
Local, Test, Perf: an arbitrary time in the distant past. It does not represent the actual latest Terra ToS release in these environments.